### DISCUSSION TITLE
Clicking in something

### DISCUSSION THREAD

**Discussion by TimDrms** [ORelio/Minecraft-Console-Client#1984] [Q&A]
Hello, when I connect to the server where I want to afk I need to click somewhere in the screen (in one of the 4 items, no matter which one).
Do you know if there is a way to do that with MCC or not ?
<img width="299" alt="image" src="https://user-images.githubusercontent.com/92984690/161232929-9e0cda19-2478-4153-bfdc-2fbda14a8c9b.png">

:)

**Comment by ORelio** [comment]
`/inventory container click <slot>`. Need to enable inventory handling in config. Not supported for every Minecraft version.
